web/client/epics/__tests__/queryparam-test.js

```diff
@@ -761,7 +761,7 @@ describe('queryparam epics', () => {
             },
             router: {
                 location: {
-                    search: "?featureInfo=38.72,-95.625"
+                    search: "?featureInfo=-95.625,38.72"
```

Comment:
I applied this change because lat -95 doesn't exist.

Reply:
Thanks ;)